test/test.py

```diff
@@ -277,6 +278,8 @@ class OSBuild(contextlib.AbstractContextManager):
     _exitstack = None
     _cachedir = None
 
+    maximum_cache_size = 20 * 1024 * 1024 * 1024  # 20 GB
+
```

Comment:
I think we should eventually configure that from the CI, rather than in the tests. I mean, for direct objectstore unit-tests we can configure it, but for everything else I guess we would prefer if the cache was properly pre-populated and cache elsewhere. But that is something for the future to figure out.

Reply:
I totally agree!